app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -195,6 +196,9 @@ class LuciBuildService {
           userData: base64Encode(json.encode(userData).codeUnits),
         ),
         fields: 'id,builder,number,status,tags',
+        exe: <String, dynamic>{
+          'cipdVersion': pullRequest.base!.ref!,
```

Comment:
By `use the recipes version that corresponds to the PR base ref on presubmit.`, how do we create/maintain the mapping between the recipes version and the pull request base ref?

Reply:
The branch in recipes will be created whenever a new beta branch is created. The process is really simple three commands and aligning a commit from flutter/flutter to a commit in the recipes folder. If a release branch does not have a corresponding release branch in recipes it will use the latest version from master.

The automation to bundle recipes currently bundles changes to master, main and flutter-* branches.